crates/oxc_cli/src/command.rs

```diff
@@ -139,15 +139,15 @@ mod test {
         let arg = "oxc lint foo.js";
         let matches = Command::new().build().try_get_matches_from(arg.split(' ')).unwrap();
         let matches = matches.subcommand_matches("lint");
-        assert_eq!(matches.unwrap().get_one::<usize>("max-warnings"), None)
+        assert_eq!(matches.unwrap().get_one::<usize>("max-warnings"), None);
```

Comment:
`cargo lint` seems to flag these in my recent PRs, must have force merged changes without the lint check. We can probably ignore until they're fixed in `main`

Reply:
Main is fixed.